bci_tester/data.py

```diff
@@ -58,19 +58,36 @@
     )
 
 
-OS_MAJOR_VERSION, OS_SP_VERSION = (int(ver) for ver in OS_VERSION.split("."))
+if OS_VERSION == "tumbleweed":
+    OS_MAJOR_VERSION = 17
+    OS_SP_VERSION = 0
```

Comment:
The 17 feels kinda arbitrary, why not just `tumbleweed`? Or the year?

Reply:
its the current suse version (sle 16 is skipped)

Reply:
TIL